Index: data/casino.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\n    \"balance\": 43711,\n    \"inventory\": [\n        {\n            \"index\": 5,\n            \"type\": \"Hamster\",\n            \"value\": 240\n        },\n        {\n            \"index\": 9,\n            \"type\": \"Elephant\",\n            \"value\": 5029\n        },\n        {\n            \"index\": 2,\n            \"type\": \"Giraffe\",\n            \"value\": 1020\n        }\n    ]\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/data/casino.json b/data/casino.json
--- a/data/casino.json	
+++ b/data/casino.json	
@@ -1,20 +1,4 @@
 {
-    "balance": 43711,
-    "inventory": [
-        {
-            "index": 5,
-            "type": "Hamster",
-            "value": 240
-        },
-        {
-            "index": 9,
-            "type": "Elephant",
-            "value": 5029
-        },
-        {
-            "index": 2,
-            "type": "Giraffe",
-            "value": 1020
-        }
-    ]
+    "balance": 0,
+    "inventory": []
 }
\ No newline at end of file
